### PR TITLE
add some CMS and Apollo-Router documentation

### PR DESCRIPTION
**Pull request by nikomakela**
HCRC-82

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-hobbies-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-events-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by nikomakela** [comment]
Added a full list of reviewers so the new team members can copy the list to their own upcoming pull requests 😄

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-sports-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr230.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr230.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr230.dev.hel.ninja :laughing::tada::tada::tada:

File: apps/events-helsinki/README.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+### Route (pages)
+
+The pages are served with some server side rendering (SSR) mechanism to offer better search engine optimization (SEO) and fast user friendly UI.
+The pre-rendering process that we mostly use here is SSG - "automatically generated as static HTML + JSON (uses getStaticProps)".
+More about NextJS's data fetching in https://nextjs.org/docs/basic-features/data-fetching/overview.
+
+There are some rules that a content manager must follow while maintaining the CMS:
+
+1. **/ (The root path / the front page)**: The root path is set as a landing page and it acts as a home page or the front page. The slug for this page must be an empty string, so the uri is `/` in every language.
+2. **/search (The search page)**: The search page gets no content from the CMS, except the SEO information. The uri for this page must be `/search` in every language. It is translated to different languages with the `i18nRoutes.config.json` configuration file.
```
Comment (nenonja) [inline apps/events-helsinki/README.md:35]: There could be a straight link to the `i18nRoutes.config.json` file

Comment (nenonja) [reply apps/events-helsinki/README.md:35]: In same style as `[README](./proxies/events-graphql-federation/README.md)`

File: apps/hobbies-helsinki/README.md [hunk 2]
```diff
@@ -8,16 +8,68 @@ Production server: https://harrastukset.hel.fi
...
+The page will reload if you make changes.
+
+###`yarn build`
+
+Builds the production application in the `.next` folder.
+Production build can be run locally with `yarn start`.
+
+###`yarn test`
+Launches the test runner in the interactive watch mode.
+
```
Comment (nenonja) [inline apps/hobbies-helsinki/README.md:72]: There is lot of other scripts that could be also listed like lint, typecheck, etc.

**Review by nenonja — APPROVED** [review]
Some minor comments, but can be merged too as it's only a doc

File: apps/events-helsinki/README.md (hunk 1 above)
Comment (nikomakela) [reply apps/events-helsinki/README.md]: added

File: apps/hobbies-helsinki/README.md (hunk 2 above)
Comment (nikomakela) [reply apps/hobbies-helsinki/README.md:73]: added some

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-hobbies-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-sports-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-events-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr230.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr230.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr230.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-hobbies-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-sports-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr230.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr230.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr230.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr230.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**